src/common/operationEntityAttributes.ts

```diff
@@ -0,0 +1,80 @@
+import { DeliverableAnalyzerOperation } from 'pnc-api-types-ts';
+
+import { TEntityAttributes } from 'common/entityAttributes';
+
+// this supertype is not coming from pnc-api-types-ts
```

Comment:
muti-line comments are suggested:
`/*
 * This supertype is not coming from pnc-api-types-ts,
 * but these props are the same for all Operations.
 */`